### ISSUE TITLE
DRC now fails

### ISSUE THREAD

**Issue by thejpster** [Neotron-Compute/Neotron-Pico#140]
Something changed in a recent version of the kibot container, and now DRC fails.

```
$ docker run --rm -ti -v $(pwd):/work setsoft/kicad_auto:ki6 bash -c "cd /work/Kicad && kibot -c docs.kibot.yml -e neotron-pico.kicad_sch -b neotron-pico.kicad_pcb -d docs"
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
- Updating BoM in XML format
WARNING:(W058) Missing KiCad main config file /root/.config/kicad/6.0/kicad_common.json (eeschema_do - misc.py:181) (kibot - kiplot.py:128)
WARNING:(W043) Missing component `Neotron-Common-Hardware:Double-mini-DIN-6` (kibot - v6_sch.py:1930)
- Running the ERC
WARNING:(W058) 2 ERC warnings detected (eeschema_do - eeschema_do:819) (kibot - kiplot.py:128)
WARNING:(W058) (multiple_net_names) Both +3V3 and VCC are attached to the same items; +3V3 will be used in the netlist

    @(195.58 mm, 59.69 mm): Symbol #PWR0303 [+3V3] Pin 1 [+3V3, Power input, Line]
    @(219.71 mm, 147.96 mm): Symbol U303 [74AHC1G04] Pin 5 [VCC, Power input, Line] (eeschema_do.kiauto.file_util - file_util.py:149) (kibot - kiplot.py:128)
WARNING:(W058) (lib_symbol_issues) Symbol 'Double-mini-DIN-6' has been modified in library 'Neotron-Common-Hardware'.

    @(34.92 mm, 154.94 mm): Symbol J1007 [Double-mini-DIN-6] (eeschema_do.kiauto.file_util - file_util.py:149) (kibot - kiplot.py:128)
ERROR:48 ERC errors detected (eeschema_do - eeschema_do:822) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 78.74 mm): Label 'VGA_BLUE0' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 73.66 mm): Label 'VGA_GREEN2' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 76.20 mm): Label 'VGA_GREEN3' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 71.12 mm): Label 'VGA_GREEN1' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 81.28 mm): Label 'VGA_BLUE1' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 86.36 mm): Label 'VGA_BLUE3' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 83.82 mm): Label 'VGA_BLUE2' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 60.96 mm): Label 'VGA_RED1' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 68.58 mm): Label 'VGA_GREEN0' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 63.50 mm): Label 'VGA_RED2' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 66.04 mm): Label 'VGA_RED3' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(167.00 mm, 58.42 mm): Label 'VGA_RED0' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 74.93 mm): Label '~{IRQ7}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 69.85 mm): Label '~{IRQ5}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 72.39 mm): Label '~{IRQ6}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 64.77 mm): Label '~{IRQ3}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 67.31 mm): Label '~{IRQ4}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 62.23 mm): Label '~{IRQ2}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 57.15 mm): Label '~{IRQ0}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(130.56 mm, 59.69 mm): Label '~{IRQ1}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 92.71 mm): Label '~{CS5}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 90.17 mm): Label '~{CS4}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 87.63 mm): Label '~{CS3}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 85.09 mm): Label '~{CS2}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 82.55 mm): Label '~{CS1}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 80.01 mm): Label '~{CS0}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 97.79 mm): Label '~{CS7}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(208.91 mm, 95.25 mm): Label '~{CS6}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(114.30 mm, 60.96 mm): Label 'DIGITAL_IN1' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(139.70 mm, 60.96 mm): Label 'DIGITAL_IN2' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(165.10 mm, 60.96 mm): Label 'DIGITAL_IN3' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(88.90 mm, 60.96 mm): Label 'DIGITAL_IN0' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(114.30 mm, 60.96 mm): Label 'DIGITAL_IN1' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(139.70 mm, 60.96 mm): Label 'DIGITAL_IN2' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(165.10 mm, 60.96 mm): Label 'DIGITAL_IN3' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(88.90 mm, 60.96 mm): Label 'DIGITAL_IN0' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(114.30 mm, 60.96 mm): Label 'DIGITAL_IN1' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(139.70 mm, 60.96 mm): Label 'DIGITAL_IN2' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(165.10 mm, 60.96 mm): Label 'DIGITAL_IN3' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(88.90 mm, 60.96 mm): Label 'DIGITAL_IN0' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(113.67 mm, 168.28 mm): Label '~{CS7}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(113.67 mm, 170.81 mm): Label '~{CS6}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(71.75 mm, 168.28 mm): Label '~{IRQ7}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(71.75 mm, 170.81 mm): Label '~{IRQ6}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(35.56 mm, 49.53 mm): Label '~{CS7}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(35.56 mm, 52.07 mm): Label '~{CS6}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(50.80 mm, 49.53 mm): Label '~{IRQ7}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:(label_dangling) Label not connected to anything

    @(50.80 mm, 52.07 mm): Label '~{IRQ6}' (eeschema_do.kiauto.file_util - file_util.py:140) (kibot - kiplot.py:125)
ERROR:ERC errors: 48 (kibot.pre_run_erc - pre_run_erc.py:74)
```

See also https://github.com/Neotron-Compute/Neotron-Pico/actions/runs/3429460913/jobs/5715107967

**Comment by thejpster** [comment]
The same errors appear in Kicad 6.0.9 for macOS. They did not appear in 6.0.6 for macOS.

**Comment by thejpster** [comment]
Pinning to 6.0.7 as a workaround.

Seems to be this issue: https://gitlab.com/kicad/code/kicad/-/issues/12814

**Comment by thejpster** [comment]
Fixed upstream, waiting for KiCAD 6.0.10 release.

**Comment by thejpster** [comment]
Fixed in KiCAD 6.0.10 which is now out. Need updated kibot container.

**Comment by thejpster** [comment]
KiCAD 6.0.10 crashes when you run ERC and apparently there won't be a 6.0.11 (see https://gitlab.com/kicad/code/kicad/-/issues/13254). So let's stick with 6.0.7.
